Remove version pins - we have newer version of both
components and they both seem 100% compatible.

Not intended for upstream.

--- oci-2.164.2/pyproject.toml
+++ oci-2.164.2/pyproject.toml
@@ -27,8 +27,8 @@ classifiers=[
 ]
 dependencies = [
     "configparser==4.0.2 ; python_version < '3'",
-    "cryptography>=3.2.1,<46.0.0",
-    "pyOpenSSL>=17.5.0,<=25.1.0",
+    "cryptography",
+    "pyOpenSSL",
     "python-dateutil>=2.5.3,<3.0.0",
     "pytz>=2016.10",
     "circuitbreaker>=1.3.1,<2.0.0; python_version <= '3.6'",
